Dockerfile

```diff
@@ -1,11 +1,25 @@
-FROM python:3.10.4-slim
+ARG  dockerfile_from_image=europe-west2-docker.pkg.dev/ons-ci-rm/docker/python-pipenv:latest
+FROM ${dockerfile_from_image} as build
```

Comment:
Worth giving the build arg a more descriptive name, since this Dockerfile has multiple build stages with different base images, something like `python_pipenv_build_image` to make it clear what image this arg should point to?